.github/workflows/develop-rebase.yml

```diff
@@ -0,0 +1,21 @@
+name: Develop Rebase
+on:
+  push:
+    branches:
+      # All pushes to this branch will trigger the task.
+      - master
+
+jobs:
+  rebase_branches:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: develop
+      - name: Rebase develop on master
+        run: |
+          git fetch --all
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git rebase --autosquash --autostash origin/master develop
```

Comment:
wont --autosquash squash the development commits?

Reply:
From the man git-rebase page:

--autosquash
When the commit log message begins with "squash! ..." (or "fixup! ..."), and there is a commit whose title begins with the same ..., automatically modify the todo list of rebase -i so that the commit marked for squashing comes right after the commit to be modified, and change the action of the moved commit from pick to squash (or fixup).